css/index.css

```diff
@@ -112,6 +130,46 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes .box:first-child {
+    background-color: teal;
+}
+
+.middle-content .boxes .box:nth-child(2) {
+    background-color: gold;
+}
+
+.middle-content .boxes .box:nth-child(3) {
+    background-color: cadetblue;
+}
+
+.middle-content .boxes .box:nth-child(4) {
+    background-color: coral;
+}
+
+.middle-content .boxes .box:nth-child(5) {
+    background-color: crimson;
+}
```

Comment:
Awesome, you got it working. Ideally, however, you want to stay away from going in too deep in terms of the CSS parents and child. A way to achieve the same results with different colored boxes is to target the parent directly and alter its child. For example:

```
.boxes .box:first-child{

}
```

You can essentially remove the ".middle-content" level.